textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/Span.java

```diff
@@ -11,14 +11,21 @@
     private String key;
     //The value matching the key
     private String value;
-    
-    
-    public Span(String fieldName, int start, int end, String key, String value) {
+    // The token position of the sapn
+    private int tokenOffset;

```

Comment:
Since we added one more offset, it will be good add comments to explain that "start" and "end" are character offsets.  Also add an example to explain their meaning, and explain that character offsets are for "gaps," and "tokenOffset" starts from 0 (?).